addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -30,7 +30,7 @@ def default_get(self, field_list):
             result['encoding_uom_id'] = self.env.company.timesheet_encode_uom_id.id
```

Comment:
I would guess you can also add the `encoding_uom_id` only if it is a timesheet

Reply:
IMO, those lines are indeed useless since it is a computed field not stored and so the compute method will be trigger when the field is visible in the view.

So, indeed you could directly remove those lines. 🙂